docs/_includes/fullscreen-dialog.html

```diff
@@ -1,6 +1,7 @@
 <div id="fullscreen-dialog">
     {% include section-header.html name="fullscreen-dialog" version=page.module_metadata.fullscreen-dialog.ds-component.version %}
 
+    <p>** DEPRECATED ** Due to non responsiveness. Use <span class="highlight">lightbox-dialog</span> with <span class="highlight">lightbox-dialog--large</span> instead </p>
```

Comment:
Since this is a major version, we should consider removing this modal completely. We can add the deprecation and point to `lightbox-dialog` in the release notes.

Reply:
I think for now we will keep these in because we did not give teams time to remove them.